src/uu/mv/src/mv.rs

```diff
@@ -441,7 +509,46 @@ fn rename_with_fallback(from: &Path, to: &Path) -> io::Result<()> {
                 copy_inside: true,
                 ..DirCopyOptions::new()
             };
-            if let Err(err) = move_dir(from, to, &options) {
+
+            // Calculate total size of directory
+            // Silently degrades:
+            //    If finding the total size fails for whatever reason,
+            //    the progress bar wont be shown for this file / dir.
+            //    (Move will probably fail due to permission error later?)
+            let total_size = match dir_get_size(from) {
+                Ok(size) => Some(size),
+                Err(_) => None,
+            };
+
+            let progress_bar = if b.progress_bar {
+                if let Some(total_size) = total_size {
+                    let bar = ProgressBar::new(total_size).with_style(
+                        ProgressStyle::with_template(
+                            "{msg}: [{elapsed_precise}] {wide_bar} {bytes:>7}/{total_bytes:7}",
+                        )
+                        .unwrap(),
+                    );
+
+                    match multi_progress {
+                        Some(mp) => Some(mp.add(bar)),
+                        None => Some(bar),
+                    }
```

Comment:
I think we can simplify this logic, by always making a `MultiProgress` if `b.progress_bar` is `true`. Just with either zero or one bar already in it. Then we only have to match on `multi_progress` and check `b.progress_bar` anymore.